python/social_bot/models/pr2_noplugin/model.sdf

```diff
@@ -5,7 +5,7 @@
     <link name="base_footprint">
       <self_collide>0</self_collide>
       <inertial>
-        <mass>118.001000</mass>
+        <mass>50.0</mass>
```

Comment:
I don't know what other people typically use here, if 50 is not a typical number, maybe add a comment to it explaining why it's changed and what's the default number is?

Reply:
Check this:
http://ode.org/ode-latest-userguide.html#sec_8_1_0
we are using FastStep(WorldStep is several ten times slower). 
>  When a joint connects two bodies with a large mass ratio (i.e. one body has several times the mass of the other body) StepFast1 may have trouble solving it.
> 
> ...
> 
> Noting the mass ratio problem above, you might want to consider making the mass of every body in your system equal to 1.0. Or in a very small range, for example between 0.5 and 1.5. Most of the other suggestions for speed and stability apply to StepFast1, except that the object is no longer to remove as many joints as possible from the equation.
> 

and 
http://ode.org/ode-latest-userguide.html#sec_11_4_0
> mass ratios - e.g. a whip. Joints that connect large and small masses together will have a harder time keeping their error low. 

To be simple, Because a higher mass ratio between 2 joint/contacts constrained links (e.g., > 10 or < 0.1) makes sim accuracy lower, could cause instability problem and makes LCP solver needs more iterations to converge in 1 sim step.

The PR2 model use the real mass of the physical model. mass between the links various a lot. Make the ratio lower increase the accuracy and stability and does not affect the dynamics. 
So high mass objects are set lower and light objects are made heavier.